### PR TITLE
#58 created test

### PR DESCRIPTION
**Pull request by apocarteres**

### CODE REVIEW

**Comment by dmarkov** [comment]
I'll find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by dmarkov** [comment]
@krzyk please review this one

**Comment by krzyk** [comment]
@apocarteres build on travis fails

**Comment by apocarteres** [comment]
@krzyk right, but it's not connected with my changes. take a look at logs

**Comment by krzyk** [comment]
@apocarteres but with those tests failing I can't be sure that your changes pass qulice checks.
You could report a bug for this project and wait for someone to fix it.

**Comment by apocarteres** [comment]
@krzyk raised a bug. restarted build

**Comment by krzyk** [comment]
@apocarteres you can now add `@Ignore` to the tests that are failing and add a todo to enable them when the bug you created is fixed

**Comment by apocarteres** [comment]
@krzyk looks like a curse :) Travis killed the build...

**Comment by krzyk** [comment]
@apocarteres so let's try again :), close and reopen this PR, this will force travis to make another build

**Comment by apocarteres** [comment]
closing down this PR to reopen new one (to force Travis build project again)

**Comment by krzyk** [comment]
@apocarteres no no, just reopen this one, it is enough

**Comment by apocarteres** [comment]
@krzyk any thoughts on that? does it more likely a Travis problem?

**Comment by krzyk** [comment]
@apocarteres I have no idea why this might happen, does it happen without your changes also?

**Comment by apocarteres** [comment]
yes it does, since i have ignored my test and not only that one

Best regards, 
Software engineer, Paderin Alexander

Please, feel free to contact me via: 

skype: lyrtenor
cell: +7(916) 415-1928
email: apocarteres@gmail.com

**Comment by krzyk** [comment]
@apocarteres in that case, please report a bug in this project and we will have to wait for it to be resolved

**Comment by apocarteres** [comment]
@krzyk i have raised one 3 days ago #63

File: src/test/java/com/jcabi/mysql/maven/plugin/ConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+ * <code>@Ignore</code> must be removed from this test and
+ * other with todo #63
+ * @author Alexander Paderin (apocarteres@gmail.com)
+ * @version $Id$
+ * @checkstyle ClassDataAbstractionCoupling (170 lines)
+ * @checkstyle MultipleStringLiterals (170 lines)
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+@Ignore
+public final class ConfigTest {
```
Comment (krzyk) [inline src/test/java/com/jcabi/mysql/maven/plugin/ConfigTest.java]: this test should be named `ConfigITCase` (it's an integration test)

Comment (krzyk) [inline src/test/java/com/jcabi/mysql/maven/plugin/ConfigTest.java]: moreover it should be in `src/it/` directory, please see other tests there

**Comment by krzyk** [comment]
@apocarteres my comments above, more over this test should be created in similar way as `WithOptionsITCase`, as an integration test. So you should create a directory `with-config` inside `src/it/` directory and create a project that uses `config` parameter in pom.xml

**Comment by dmarkov** [comment]
@krzyk thanks, I just added **23 mins** to your account, payment `52104150`

extra minutes for review comments (c=8)

+23 to your rating, your total score is [+4492](http://www.netbout.com/b/35035?open=rating)

**Pull request closed without merging**